PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -546,6 +537,15 @@ if [ "$CustomBrain" = "NONE" ] ; then
       fi
       OutputTXwImageSTRING=""
 
+      # skip modality if no image
+
+      if [ "${TXwInputImages}" = "NONE" ] ; then
+        log_Msg "Skipping Modality: " $TXw " - image not specified."
+        continue
+      else
+        log_Msg "Processing Modality: " $TXw
```

Comment:
Unrelated, but this just looks weird to me.  Can we put the entire `log_Msg`'s inside a single quotation mark?
e.g.,
```
      if [ "${TXwInputImages}" = "NONE" ] ; then
        log_Msg "Skipping Modality: $TXw - image not specified."
        continue
      else
        log_Msg "Processing Modality: $TXw"
```

Reply:
I missed that - I agree, please change the messages to Mike's suggestion, to make it obvious how spaces get preserved (the existing version will result in double spaces around the value of TXw).

Reply:
Changed.